tutorials/tutorial4_certified_over_estimation.ipynb

```diff
@@ -0,0 +1,735 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "82f17f7b",
+   "metadata": {},
+   "source": [
+    "## Landing Distance Estimation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aac59c5f",
+   "metadata": {},
+   "source": [
+    "In many cases, **safety can be ensured by systematically over estimating** the reference model. An intuitive example\n",
+    "is a surrogate that would predict the landing distance of an aircraft. Under-estimating the reference  distance  could  lead  to  an  avoidable  overrun. On  the contrary, over-estimating the reference distance may lead to unnecessary turn-around maneuvers when the landing was indeed safely possible.\n",
+    "\n",
+    "**Based on this reasonning we can define safety as the following requirement: the neural network will over-estimate the reference model on any point**\n",
+    "\n",
+    "<p align=\"center\">\n",
+    "  <img src=\"./usecase/img/safe_surrogate.png\" alt=\"SafeOverestimation!\" width=\"400\">\n",
+    "</p>\n",
+    "\n",
+    "In this notebook, we will demonstrate how to _**train and assert with that a neural network is a safe surrogate**_."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8e227ae3",
+   "metadata": {},
+   "source": [
+    "### UseCase Description\n",
+    "\n",
+    "The [Cessna 172 Skyhawk](https://en.wikipedia.org/wiki/Cessna_172) is an American four-seat, single-engine, high wing, fixed-wing aircraft made by the Cessna Aircraft Company. Measured by its longevity and popularity, the Cessna 172 is the most successful aircraft in history. Cessna delivered the first production model in 1956, and as of 2015, the company and its partners had built more than 44,000 units.The aircraft remains in production today.\n",
+    "\n",
+    "<p align=\"center\">\n",
+    "  <img src=\"usecase/img/cesna_c_172.jpeg\" alt=\"SafeOverestimation!\" width=\"400\">\n",
+    "</p>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c7f8b227",
+   "metadata": {},
+   "source": [
+    "## Hands on ! Getting the data and training a braking distance estimator"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "da211b90",
+   "metadata": {},
+   "source": [
+    "We will assess the over-estimation guanratees we can obtain for landing estimation on a toy usecase (considering only  pressure altitude and temperature).\n",
+    "You can also build your own simulator using free online calculator for CESSNA C172 (source Flight Performance App)\n",
+    "\n",
+    "### Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "62b410e4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "\n",
+    "sys.path.append(\"./usecase\")\n",
+    "import matplotlib.patches as patches\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from cesna_simulation import cesna_landing\n",
```

Comment:
Does not work in colab because the notebook is "alone" the modules around are not downloaded on the colab machine by default. But we can tackle this in another PR. In binder this should work as all files from the repo will be there.